packages/core/src/interfaces/charts.ts

```diff
@@ -602,8 +601,8 @@ export interface ChoroplethChartOptions extends ThematicChartOptions {
 			title?: string
 			type: ColorLegendType | string
 		}
-		geoData: Topology
```

Comment:
This was a bug. The prop `geoData` was expected to be at the top-level of `ChoroplethChartOptions` - not in the `choropleth` property. If you try to set choropleth.geoData, it does not work.

The original demo for Choropleth set the `geoData` at the top-level which is how I found this bug.